platformio.ini

```diff
@@ -95,7 +96,7 @@ board_build.extra_flags = '-DEMSESP_DEFAULT_BOARD_PROFILE="S32"'
 
```

Comment:
The 4M version could be S32 or E32, for this we need the E32 test.

Reply:
No, the EMSESP_DEFAULT_BOARD_PROFILE is stored in code. If an E32 user updates the 4M version and make factory reset it will **now** default to Wifi-S32 instead of searching the ethernet. That's why the test code is included and 4M should have board_profile "default".

Reply:
so is the best fix here is to not set a EMSESP_DEFAULT_BOARD_PROFILE in pio for `ci_s_4M` and let the code run the 'default' logic?

Reply:
> so is the best fix here is to not set a EMSESP_DEFAULT_BOARD_PROFILE in pio for ci_s_4M and let the code run the 'default' logic?

Yes.